wadl/lib/route.py

```diff
@@ -167,6 +175,40 @@ def DistGPS(gps0, gps1, alt0=0, alt1=0):
         e1, n1, _, _ = utm.from_latlon(*gps1)
         return np.linalg.norm([e0-e1, n0-n1, alt0-alt1])
 
+    def approx(self, a, b):
+        # accepts 2 utm points and return true if they are approximately the same value
+        if abs(a[0]-b[0]) < 0.1 and abs(a[1]-b[1]) < 0.1:
+            return True
+        return False
+
+    def setMaze(self, maze):
+        self.maze = maze
+
+    def unstreamline(self, metaGraph, i):
```

Comment:
i see what you did here, but man we need a way better way to manage all these abstractions of routes